controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -61,7 +67,7 @@ func TestNoMemberStatusFound(t *testing.T) {
 		// given
 		requestName := "bad-name"
 		getHostClusterFunc := newGetHostClusterReady
-		reconciler, req, _ := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl)
+		reconciler, req, _ := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl, mockLastGitHubAPICall, test.MockGitHubClientForRepositoryCommits(buildCommitSHA, time.Now().Add(-time.Hour*1)))
```

Comment:
You could do the same as in host-operator PR https://github.com/codeready-toolchain/host-operator/pull/819#discussion_r1243552199

Reply:
I've cleaned it up a bit . Pls see [78a30e1](https://github.com/codeready-toolchain/member-operator/pull/446/commits/78a30e10315b864af392e7a11c435de06468920a) and [daee1ad](https://github.com/codeready-toolchain/member-operator/pull/446/commits/daee1ad8962b8530c1da5e096e0f3ac12a8b7a1d)